docs/content/docs/troubleshooting/traffic/ingress.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Ingress Troubleshooting"
+description: "Ingress Troubleshooting Guide"
+type: docs
+aliases: ["ingress.md"]
+---
+
+## When Ingress is not working as expected
+
+### 1. If using HTTPS ingress, confirm HTTPS ingress is enabled on OSM configmap
+
+Because use_https_ingress is disabled by default, this variable needs to be updated in the OSM configmap to enable HTTPS ingress on the cluster. 
+
+```console
+# Returns true if HTTPS ingress is enabled
+$ kubectl get ConfigMap osm-config -n osm-system -p jsonpath='{.data.use_https_ingress}{"\n"}'
+true
+```
+
+If HTTPS ingress is not enabled, you can enable it with the following command: 
+
+> Note: To make this change persistent between upgrades, see osm mesh upgrade --help.
+```bash
+# Replace osm-system with osm-controller's namespace if using a non default namespace
+kubectl patch ConfigMap osm-config -n osm-system -p '{"data":{"use_https_ingress":"true"}}' --type=merge
+```
+
+The above command must return a boolean string (`true` or `false`) indicating if egress is enabled.
+
+### 2. Inspect OSM controller logs for errors
+
+
+```bash
+# When osm-controller is deployed in the osm-system namespace
+kubectl logs -n osm-system $(kubectl get pod -n osm-system -l app=osm-controller -o jsonpath='{.items[0].metadata.name}')
+```
+
+Errors will be logged with the `level` key in the log message set to `error`:
+```console
+{"level":"error","component":"...","time":"...","file":"...","message":"..."}
+```
+
+### 3. Confirm that the ingress resource has been successfully deployed
+
+```bash
+kubectl get ingress <ingress-name> -n <ingress-namespace>
+```
+
+### 4. Confirm the Envoy configuration
```

Comment:
Let's remove this since content is missing for this section. We can add it later.